packages/base/src/SVGIconRegistry.js

```diff
@@ -14,9 +14,9 @@ const calcKey = (name, collection) => {
 	return `${collection}:${name}`;
 };
 
-const registerIcon = (name, { pathData, accData, collection } = {}) => {
+const registerIcon = (name, { pathData, ltr, accData, collection } = {}) => { // eslint-disable-line
```

Comment:
Why is the eslint ignore?

Reply:
I get the following:
17:29  error  Expected a line break after this opening brace   object-curly-newline

Reply:
I am getting these (although I am adding one additional key): 
 17:29  error  Expected a line break after this opening brace   object-curly-newline
 17:66  error  Expected a line break before this closing brace  object-curly-newline